phpBB/includes/acp/acp_board.php

```diff
@@ -1047,6 +1048,35 @@ function board_disable($value, $key)
 		return h_radio('config[board_disable]', $radio_ary, $value) . '<br /><input id="' . $key . '" type="text" name="config[board_disable_msg]" maxlength="255" size="40" value="' . $this->new_config['board_disable_msg'] . '" />';
 	}
 
+	/**
+	 * Board disable access for which group: admins: 0; plus global moderators: 1 and plus all moderators: 2
+	 */
+	public function board_disable_access($value, $key = '') : array
+	{
+		global $user;
+
+		return [
+			[
+				'tag' 		=> 'select',
+				'value'		=> 0,
+				'selected'	=> ($value == 0) ? ' selected="selected"' : '',
```

Comment:
IIRC it's just a boolean switch in macros, so `'selected'	=> $value == 0` should be enough. Same below.

Reply:
Yes, i changed it :)